infra/nightly-resources/index.js

```diff
@@ -126,34 +132,50 @@ function groupByBenchmark(benchList) {
     return groupedBy;
 }
 
+// b may be undefined
+function compareTo(a, b) {
+    // if b is undefined, return a
+    if (b === undefined) {
+        return tryRound(a);
+    } else {
+        var difference = a - b;
+        var sign = difference < 0 ? "" : "+";
+        // put the difference in parens after a
+        return tryRound(a) + " (" + sign + tryRound(difference) + ")";
```

Comment:
Not sure how I feel about this implicit type conversion. Can we use a format string?

Reply:
+1